File: Encuentro #01/C#/Ejercicio_01/Ejercicio_01/Participante.cs
using System;

namespace Ejercicio_01
{
    public class Participante
    {
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Email { get; set; }
        
        public Participante() { }
        
        public Participante(string nombre, string apellido, string email)
        {
            Nombre = nombre;
            Apellido = apellido;
            Email = email;
        }
        
        public void LeerParticipante()
        {
            Console.WriteLine("Rellene los Datos Personales : ");

            Console.Write("Nombre: ");
            Nombre = Console.ReadLine();

            Console.Write("Apellido: ");
            Apellido = Console.ReadLine();

            Console.Write("Ingrese su Email: ");
            Email = Console.ReadLine();
        }
        
        protected void MostrarParticipante()
        {
            Console.WriteLine($"DATOS DEL PARTICIPANTE\nNombre: {Nombre}\nApellido: {Apellido}\nEmail: {Email}");
        }
        
    }
}

File: Encuentro #01/Java/Ejercicio 01/src/com/pplam/modelos/Participante.java
package com.pplam.modelos;
import java.util.Scanner;

public class Participante {
    private String nombre, apellido, email;

    public Participante() {}

    public Participante(String nombre, String apellido, String email) {
        this.setNombre(nombre);
        this.setApellido(apellido);
        this.setEmail(email);
    }

    public void leerParticipante() {
        System.out.println("Nombre: ");
        this.setNombre(new Scanner(System.in).nextLine());

        System.out.println("Apellido: ");
        this.setApellido(new Scanner(System.in).nextLine());

        System.out.println("Email: ");
        this.setEmail(new Scanner(System.in).nextLine());
    }

    protected void mostrarParticipante() {
        System.out.print("Nombre: " + this.getNombre() + "\nApellido: " + this.getApellido() + "\nEmail: " + this.getEmail());
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }
}
